### PR TITLE
quest checks

### PR DESCRIPTION
**Pull request by robertlabrie**

### CODE REVIEW

File: LazyCraftingCertification.lua [hunk 1]
```diff
@@ -70,8 +80,9 @@ local function LCCDeconstructFirstMatchingBackpackItem(q)
 end
 
 function LazyCraftingCertification.CraftingStation(eventCode, craftSkill, sameStation)
-  local isQuest,questStep = LCCCheckCertificationQuest()
+  local isQuest,questStep,questCraftingType = LCCCheckCertificationQuest()
```
Comment (Dolgubon) [inline LazyCraftingCertification.lua:83]: So, you get a singular question craft type here, and that craft type is independent of the actual station you are currently at. This means that, if you have multiple certs active, only one will be progressed by the addon. (Whichever one has the lowest journal quest index) 
So the search has to be improved to either the status of the cert for the station you are at or the status of all certs in a table.

Comment (robertlabrie) [reply LazyCraftingCertification.lua:83]: Is it possible to have multiple cert quests active though? Danel and Millenith will only give you one at a time if you try talking to them before completing your active quest they just repeat the instructions. I guess it's possible to pick up one from Danel and one from Millenith and have 2 active at the same time. I'll try it out and see but if not then I think this is fine.

Yes other option is to build a table where key is the questCraftingType and the value is boolean and then lookup in the table based on the CRAFTING_STATION_TYPE when visited.

Comment (Dolgubon) [reply LazyCraftingCertification.lua:83]: Oh, didn't realize it was not possible :) Used to it being possible from writs I guess
I'm not sure about asking from different quest givers.

Comment (robertlabrie) [reply LazyCraftingCertification.lua:83]: I just confirmed that Millenith won't give a quest if you have a Danel quest active so I'm gonna go ahead and ship it. Thanks again for all your feedback and advice and for everything you do  for the ESO community.

**Pull request merged**